### DISCUSSION TITLE
How to remove/reset the data and folder of bookmark?

### DISCUSSION THREAD

**Discussion by yamakazi123** [alefragnani/vscode-bookmarks#501] [Q&A] [answered]
HI All:
When I am in folder A, the sidebar of the bookmark shows nothing. Even I add new bookmark, the sidebar still can't show anything. But if I move to folder B, the sidebar works normal. I guess something is wrong or broken in my folder A. Could anyone knows where is the bookmark extension stores the bookmark data and how could I remove/reset it?
![Uploading bookmark.JPG…]()

**Comment by alefragnani** [comment, marked as answer]
Hi @yamakazi123 ,

The bookmarks are stored in VS Code internals, which in the end, is a SQLite database. Previously it was available in `Developer Tools`, because it is in fact, the Chromium's Local Storage. More details here: https://github.com/alefragnani/vscode-bookmarks/issues/242#issuecomment-594999154

You can also make the bookmarks to be stored _inside the project_, using the `bookmarks.saveBookmarksInProject` setting. It will save the bookmarks in `.vscode/bookmarks.json` file.

To _reset_ the bookmarks, you could use the `Bookmarks: Clear from All Files` command. It should wipe out any bookmark you have, on the active session/folder.

Hope this helps

ps: the image you attached, isn't available.

**Reply by yamakazi123** [reply]
@alefragnani 
Thanks, a lot. It works. I clear the bookmark from all files and re-launch VSCode. The bookmark now work normally in folder A. Sorry about the image but I don't why I can't finish uploading the screenshot.
